torch_brain/models/poyo.py

```
@@ -166,8 +164,9 @@ def forward(
         # latent sequence
         latent_index: TensorType["batch", "n_latent", int],
         latent_timestamps: TensorType["batch", "n_latent", float],
+        # Metadata for queries
+        session_index: TensorType["batch", int],
```

Ruff (0.15.12)

[error] Undefined name `batch`

(F821)